controllers/windowsmachine_controller_test.go

```diff
@@ -23,15 +23,6 @@ func TestIsValidMachine(t *testing.T) {
 	invalidMachine3 := mapi.Machine{}
 	invalidMachine3.Name = "invalid_2"
 	invalidMachine3.Status.Phase = strToPtr("running")
-	validMachine1 := mapi.Machine{}
-	validMachine1.Name = "valid_1"
-	validMachine1.Status.Phase = strToPtr("running")
-	validMachine1.Status.Addresses = []core.NodeAddress{
-		{
-			Type:    "Hostname",
-			Address: "valid1.acme.com",
-		},
-	}
```

Comment:
While it might make sense to remove this test, this is actually a valid Machine object per the spec, so maybe we should really account for this programmatically.

Reply:
While I agree with you, I prefer opening a separate PR where we account for hostname and then reintroduce this unit test.